File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/DomainAllInfo.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Complete domain registration information returned from the registrar, including contacts, nameservers, and expiry dates.
  /// </summary>
  [DataContract]
  public class DomainAllInfo {
    /// <summary>
    /// Gets or Sets OPSVersion
    /// </summary>
    [DataMember(Name="_OPS_version", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "_OPS_version")]
    public string OPSVersion { get; set; }

    /// <summary>
    /// Gets or Sets Attributes
    /// </summary>
    [DataMember(Name="attributes", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "attributes")]
    public DomainAllInfoAttributes Attributes { get; set; }

    /// <summary>
    /// Gets or Sets _Object
    /// </summary>
    [DataMember(Name="object", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "object")]
    public string _Object { get; set; }

    /// <summary>
    /// Gets or Sets Protocol
    /// </summary>
    [DataMember(Name="protocol", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "protocol")]
    public string Protocol { get; set; }

    /// <summary>
    /// Gets or Sets ResponseText
    /// </summary>
    [DataMember(Name="response_text", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "response_text")]
    public string ResponseText { get; set; }

    /// <summary>
    /// Gets or Sets ResponseCode
    /// </summary>
    [DataMember(Name="response_code", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "response_code")]
    public string ResponseCode { get; set; }

    /// <summary>
    /// Gets or Sets Action
    /// </summary>
    [DataMember(Name="action", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "action")]
    public string Action { get; set; }

    /// <summary>
    /// Gets or Sets IsSuccess
    /// </summary>
    [DataMember(Name="is_success", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "is_success")]
    public string IsSuccess { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DomainAllInfo {\n");
      sb.Append("  OPSVersion: ").Append(OPSVersion).Append("\n");
      sb.Append("  Attributes: ").Append(Attributes).Append("\n");
      sb.Append("  _Object: ").Append(_Object).Append("\n");
      sb.Append("  Protocol: ").Append(Protocol).Append("\n");
      sb.Append("  ResponseText: ").Append(ResponseText).Append("\n");
      sb.Append("  ResponseCode: ").Append(ResponseCode).Append("\n");
      sb.Append("  Action: ").Append(Action).Append("\n");
      sb.Append("  IsSuccess: ").Append(IsSuccess).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/domain_all_info.dart
part of swagger.api;

class DomainAllInfo {
  
  String oPSVersion = null;

  DomainAllInfoAttributes attributes = null;

  String object = null;

  String protocol = null;

  String responseText = null;

  String responseCode = null;

  String action = null;

  String isSuccess = null;

  DomainAllInfo();

  @override
  String toString() {
    return 'DomainAllInfo[oPSVersion=$oPSVersion, attributes=$attributes, object=$object, protocol=$protocol, responseText=$responseText, responseCode=$responseCode, action=$action, isSuccess=$isSuccess, ]';
  }

  DomainAllInfo.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    oPSVersion = json['_OPS_version'];
    attributes = new DomainAllInfoAttributes.fromJson(json['attributes']);
    object = json['object'];
    protocol = json['protocol'];
    responseText = json['response_text'];
    responseCode = json['response_code'];
    action = json['action'];
    isSuccess = json['is_success'];
  }

  Map<String, dynamic> toJson() {
    return {
      '_OPS_version': oPSVersion,
      'attributes': attributes,
      'object': object,
      'protocol': protocol,
      'response_text': responseText,
      'response_code': responseCode,
      'action': action,
      'is_success': isSuccess
     };
  }

  static List<DomainAllInfo> listFromJson(List<dynamic> json) {
    return json == null ? new List<DomainAllInfo>() : json.map((value) => new DomainAllInfo.fromJson(value)).toList();
  }

  static Map<String, DomainAllInfo> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, DomainAllInfo>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new DomainAllInfo.fromJson(value));
    }
    return map;
  }
}
